src/main/webapp/app/exam/manage/student-exams/student-exams.component.html

```diff
@@ -43,8 +39,6 @@ <h2>
                 <fa-icon [icon]="faExclamationTriangle" class="text-warning"></fa-icon>
                 <span jhiTranslate="artemisApp.studentExams.unlockAllRepositories">Unlock all repositories</span>
             </button>
-        }
-        @if (!localVCEnabled && course?.isAtLeastInstructor) {
```

Comment:
Check for `!localVCEnabled` accidentally removed now?

Reply:
Hey, the diff is a bit confusing. If you expand the hidden part, you'll see that there's the same `@if` covering multiple elements now